src/views/simplified-account/settings/api-keys/partials/_finish-onboarding-message.njk

```diff
@@ -0,0 +1,13 @@
+{% if currentGatewayAccount.paymentProvider | lower == 'stripe' %}
+  {% set onboardingPath = paths.simplifiedAccount.settings.stripeDetails.index | formatServiceAndAccountPathsFor(currentService.externalId, currentGatewayAccount.type) %}
+  {% elseif currentGatewayAccount.paymentProvider | lower == 'worldpay' %}
+  {% set onboardingPath = paths.simplifiedAccount.settings.worldpayDetails.index | formatServiceAndAccountPathsFor(currentService.externalId, currentGatewayAccount.type) %}
+{% endif %}
+{% set sandboxModePath = paths.simplifiedAccount.enterSandboxMode.index | formatServiceAndAccountPathsFor(currentService.externalId, currentGatewayAccount.type) %}
+
+{{
+  govukInsetText({
+    html: "<a class='govuk-link govuk-link--no-visited-state' href='" + onboardingPath + "'>Finish going live</a> before you can create live API keys. <a class='govuk-link govuk-link--no-visited-state' href='" + sandboxModePath + "'>Enter sandbox mode</a> to use test API keys to connect your test digital service to GOV.UK Pay.",
```

Comment:
What happens if someone has already created API keys but has not finished onboarding?

Reply:
this should only be a very narrow subset of users (if any at all) who had created API keys while onboarding before this goes live - I could add in something to account for those users if necessary though